Extensions/nmh-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "com.3721tools.hurl",
+  "description": "Hurl",
+  "path": "E:\\Code\\Hurl\\Source\\Launcher\\target\\debug\\Launcher.exe",
```

Comment:
A hard-coded path? How does that work?

Reply:
I am still testing things out... Will deal with it later